modules/rules_powershell/0.0.3/MODULE.bazel

```diff
@@ -0,0 +1,25 @@
+"""rules_powershell"""
+
+module(
+    name = "rules_powershell",
+    version = "0.0.3",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "rules_shell", version = "0.5.1")
+bazel_dep(name = "platforms", version = "1.0.0")
+bazel_dep(name = "bazel_skylib", version = "1.8.1")
+
+bazel_dep(name = "stardoc", version = "0.8.0", dev_dependency = True, repo_name = "io_bazel_stardoc")
+bazel_dep(name = "rules_rust_mdbook", version = "0.62.0", dev_dependency = True)
+
+powershell = use_extension("//powershell:extensions.bzl", "powershell", dev_dependency = True)
```

Comment:
This means the end users have to register powershell toolchain themselves, is this intended?

Reply:
Please check https://github.com/bazel-contrib/rules-template/blob/6470c764cd1c2e122b6d0c99cf7ebd7d121b5bec/MODULE.bazel#L42-L46 and its implementation

Reply:
Do toolchains registered in an external modules behave differently than those registered in the root?